packages/@okta/vuepress-site/guides/sign-into-spa/section03/angular/attachcomponentsetc.md

```diff
@@ -1,23 +1,67 @@
-## Attach Components to Routes
-You need to provide these routes in your sample application so that we can sign the user in and handle the callback from Okta. We show you how to set these up below using [Angular Router](https://angular.io/guide/router):
+## Provide the Login and Logout Buttons
```

Comment:
This seems to be a change in focus from the original text here (explaining different context, albeit slight) - probably good to make sure we get some of the tech writers eyes here.  @susanharper-okta ^

Reply:
So - the existing Quick Start guide for Angular uses the "Attach Components to Routes" section - not the "Provide the Login and Logout Buttons" section - is this something that has changed since that guide was created? Or something that CAN change now?